compose/config/config.py

```diff
@@ -493,19 +490,14 @@ def process_config_file(config_file, environment, service_name=None):
             config_file.get_networks(),
             'network',
             environment)
-        if config_file.version in (V3_1,):
+        if config_file.version in (const.COMPOSEFILE_V3_1, const.COMPOSEFILE_V3_2):
```

Comment:
The reason I removed this line is because it adds additional steps every time we add a new version.

We should really be able to add a new version by just dropping in a schema, so I'm trying to move in that direction.

Is this check actually necessary? Don't we already check this in two places:
* first in schema validation
* again in `ConfigFile.get_secrets()`